fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PrivilegeCheckerV2.java

```diff
@@ -1539,7 +1539,7 @@ public Void visitBackupStatement(BackupStmt statement, ConnectContext context) {
             List<TableRef> tableRefs = statement.getTableRefs();
             if (tableRefs.size() == 0) {
                 String dBName = statement.getDbName();
-                throw new SemanticException("Database: {} is empty", dBName);
+                throw new SemanticException("Database: %s is empty", dBName);
```

Comment:
dbName is provided by the BACKUP statement, and it is not an optional key word for this statement. So I think it can not be empty otherwise it will cause syntax errors.